app/src/main/java/com/example/parkingsystem_kotlin/mvp/model/ParkingModel.kt

```diff
@@ -1,11 +1,13 @@
 package com.example.parkingsystem_kotlin.mvp.model
 
 import com.example.parkingsystem_kotlin.mvp.contract.ParkingContract
-import com.example.parkingsystem_kotlin.util.Constants.PARKING_SIZE
+import com.example.parkingsystem_kotlin.util.Constants
 
-class ParkingModel: ParkingContract.Model {
+class ParkingModel(private var parkingSize: Int = Constants.PARKING_SIZE) : ParkingContract.Model {
```

Comment:
you are not setting this in the constructor in the activity, you are just using it after the creation so please, put it as a variable outside the constructor

Reply:
Done.